### ISSUE TITLE
Tooltip: moving hover from tooltip back to trigger closes tooltip

### ISSUE THREAD

**Issue by jlukosky** [microsoft/fluentui#10240]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:
When moving hover from tooltip back to trigger the tooltip closes. The trigger shouldn't close the tooltip, it's behavior should be consistent. As it is the trigger is serving as a show/hide toggle.

#### Actual behavior:

Moving hover from tooltip to trigger closes tooltip.

#### Expected behavior:

Moving hover from tooltip to trigger should not close the tooltip

### Documentation describing expected behavior

The expectation is that the behavior will be consistent: hovering the trigger should show the tooltip. As it is now sometimes hovering the trigger shows the tooltip, sometimes it hides it.

**Comment by maxwellred** [comment]
Hi @jlukosky . Thanks for opening this issue about Tooltip and congratulations on this being your first in UI Fabric React! I have contacted the code owner in regards to this behavior and will be attending to it shortly.

**Comment by micahgodbolt** [comment]
Having tooltips be sticky when hovered isn't a 100% normal HTML experience, so we rely on our designers to inform us on how it should work. The fix shouldn't be too hard to do, but I want to first confirm that this is the correct design choice before doing the work.

@betrue-final-final do you have any context on if tooltip should stay open after hovereing over the tooltip and then back onto the button? @jspurlin do you know what the expectation is on the ribbon with tooltips?

**Comment by jspurlin** [comment]
I know in the desktop apps, the user can hover over the tooltip and then back over the button (but windows and native tooltips in the browser don't do that). This sounds like a feature request that should be vetted with designers.

**Comment by kenotron** [comment]
This needs spec'ing before proceeding, I'm removing myself and Max off of the assignee in this case leaving the backlog review label.

**Comment by jlukosky** [comment]
This should be reopened and resolved. If this isn't addressed it is a violation of 1.4.13: Content on Hover or Focus
